vspace/src/main/java/edu/asu/diging/vspace/web/staff/EditModuleLinkController.java

```diff
@@ -0,0 +1,75 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import edu.asu.diging.vspace.core.exception.SpaceDoesNotExistException;
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.model.display.DisplayType;
+import edu.asu.diging.vspace.core.model.display.IModuleLinkDisplay;
+import edu.asu.diging.vspace.core.services.ILinkManager;
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+
+@Controller
+public class EditModuleLinkController {
+
+    @Autowired
+    private ISpaceManager spaceManager;
+    
+    @Autowired
+    private ILinkManager linkManager;
+
+    @RequestMapping(value = "/staff/space/{id}/editmodulelink", method = RequestMethod.POST)
```

Comment:
this url looks like the id is the space id. should be `/staff/space/link/module/{id}` or something like that